extensions-core/lookups-cached-global/src/main/java/io/druid/data/input/MapPopulator.java

```diff
@@ -64,8 +64,11 @@ public long populate(final ByteSource source, final Map<K, V> map) throws IOExce
           @Override
           public boolean processLine(String line) throws IOException
           {
-            map.putAll(parser.parse(line));
-            ++count;
+            final Map<K, V> kvMap = parser.parse(line);
+            if (kvMap != null) {
+              map.putAll(kvMap);
```

Comment:
@drcrallen `MapPopulator` is used in prod only once, in `URIExtractionNamespace `. There the value is checked to be not `null`. So it seems to be fine.